packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -134,6 +134,8 @@ const String _kColorBackgroundWarning = 'Cannot provide both a backgroundColor a
 ///
 /// ![Text height comparison diagram](https://flutter.github.io/assets-for-api-docs/assets/painting/text_height_comparison_diagram.png)
 ///
+/// See [StrutStyle] for further control of line height at the paragraph level.
```

Comment:
I'm not sure how this will look inside the sample snippet with an image embedded in between text. Have you generated the docs locally to check?